assets/src/variations/expanded-link/edit.scss

```diff
@@ -0,0 +1,10 @@
+.is-style-expand-link {
+  &.is-selected a {
+    outline: 1px dashed grey;
+    outline-offset: 2px;
+  }
+
+  a:not(:first-of-type) {
```

Comment:
This is not working, since in the editor links are within another element (paragraph, heading, etc) so technically `a` tags are always `first-of-type`... I don't know how that can be solved 🤔

Reply:
Yeah I should maybe have documented this better. It's only a "best attempt" to catch some cases, like a paragraph with multiple links inside it. Not sure if it's possible with CSS only to catch the other cases.

Reply:
It would be nice to make a difference in the editor between this variation and a regular group block 🤔 Maybe the dotted outline should be around the whole block to show that it will be clickable?